### ISSUE TITLE
`no-duplicate-message' rule

### ISSUE THREAD

**Issue by kazupon** [kazupon/eslint-plugin-vue-i18n#8]

**Comment by ota-meshi** [comment]
Hi @kazupon!

I think this rule is probably better off not determining if it's a duplicate message for each locale.

For example, English and Japanese messages about dates may use the same message but have different meanings.

e.g.

```yaml
en:
  month: Month
  short_sun: S # duplicate?
  short_mon: M
  short_tue: T # duplicate?
  short_wed: W
  short_thu: T # duplicate?
  short_fri: F
  short_sat: S # duplicate?
ja:
  month: 月 # duplicate?
  short_sun: 日
  short_mon: 月 # duplicate?
  short_tue: 火
  short_wed: 水
  short_thu: 木
  short_fri: 金
  short_sat: 土
```

In my opinion, it may be possible to reduce false positives by reporting only if there are duplicates in all locales:

```yaml
en: 
  key1: dupe_message1 # Will report it
  key2: dupe_message1 # Will report it
  key3: dupe_message2 # Do not report it because it is not duplicated in "ja"
  key4: dupe_message2 # Do not report it because it is not duplicated in "ja"
ja:
  key1: 重複_メッセージ1 # Will report it
  key2: 重複_メッセージ1 # Will report it
  key3: メッセージ3
  key4: メッセージ4
```

Even then, it may still incorrectly report the same string message with different meanings.
What do you think?

**Comment by kazupon** [comment]
Indeed. :thinking: 
Regarding duplicate messages, it's probably best not to add a rule, so that there are cases where the meaning defference depending on the context of the application and the language.

In about this rule, let's keep the pending.
